.circleci/config.yml

```diff
@@ -1,5 +1,14 @@
+# CircleCI Build Config
+#   https://circleci.com/docs/2.0/configuration-reference/#section=configuration
+#
+# Targets:
+#   - Max OS/X
+#   - ARM64
+
 version: 2
 jobs:
+
+  # job: Mac OS/X raw ("build-and-test")
   build-and-test:
```

Comment:
nit, please rename this to `osx-build-and-test` to be consistent and more descriptive with the jobs names